bundles/org.openhab.binding.nanoleaf/src/main/java/org/openhab/binding/nanoleaf/internal/model/Rhythm.java

```diff
@@ -29,55 +31,55 @@
     private RhythmPos rhythmPos;
 
     public Boolean getRhythmConnected() {
-        return rhythmConnected;
+        return rhythmConnected == null ? Boolean.FALSE : rhythmConnected;
     }
 
     public void setRhythmConnected(Boolean rhythmConnected) {
         this.rhythmConnected = rhythmConnected;
     }
 
     public Boolean getRhythmActive() {
-        return rhythmActive;
+        return rhythmActive == null ? Boolean.FALSE : rhythmActive;
     }
 
     public void setRhythmActive(Boolean rhythmActive) {
         this.rhythmActive = rhythmActive;
     }
 
     public Integer getRhythmId() {
-        return rhythmId;
+        return rhythmId == null ? new Integer(0) : this.rhythmId;
```

Comment:
Why not make `rhythmId` a primitive `int`?

Reply:
Fixed